pkg/webui/containers/side-bar/side-bar.styl

```diff
@@ -0,0 +1,36 @@
+// Copyright © 2023 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.sidebar
+  border-normal(right)
+  line-height: 1
+  font-size: $fs.s
+  height: 97.5vh
```

Comment:
This is not a good solution. You should find out what causes the offset and then use `calc()` instead.